### PR TITLE
Feature style

### PR DESCRIPTION
**Pull request by Metsanpeitto**
:paintbrush:  In this branch, I have added the styles required to make the site look like the guidelines they provide to us.

### CODE REVIEW

File: src/components/Adder.js [hunk 1]
```diff
@@ -48,7 +51,7 @@ function Adder() {
           <option value="Economy">Economy</option>
           <option value="Action">Action</option>
         </select>
-        <button type="submit" onClick={submitBookToStore}>
+        <button className="btn-1" type="submit" onClick={submitBookToStore}>
           ADD BOOK
         </button>
       </div>
```
Comment (Emmyn5600) [inline src/components/Adder.js:57]: - [ ] Kindly, add padding-bottom, the add book button and placeholder seem to cover on the bottom of the pahe

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @Metsanpeitto 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Changes required :writing_hand:  

      * #### Kindly, Update the Readme file, you did some mistakes in that file.
      * #### Kindly, Remove unnecessary files in your root folder to look more clean.
      * #### Please check the comment under the review.
       
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/components/Adder.js (hunk 1 above)
Comment (Metsanpeitto) [reply src/components/Adder.js:57]: :ok_man: Thanks for your time and for calling me to save my own time. It was a nice gesture from you. :+1: 
I had been implementing every change that you required and now the App is improved.
-  Added a bottom padding.
- Updated the Readme file, you did some mistakes in that file.
- Removed unnecessary files in your root folder to look cleaner.
- Deployed the branch on gh-pages.

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**